icu4j/main/classes/core/src/com/ibm/icu/impl/units/MeasureUnitImpl.java

```diff
@@ -226,7 +222,7 @@ public void serialize() {
         }
         if (this.complexity == MeasureUnit.Complexity.COMPOUND) {
             // Note: don't sort a MIXED unit
-            Collections.sort(this.getSingleUnits(), new SingleUnitComparator());
+            this.getSingleUnits().sort(new SingleUnitComparator());
```

Comment:
I'm getting this error, which may be a Java 7 vs Java 8 thing? :

The method sort(MeasureUnitImpl.SingleUnitComparator) is undefined for the type ArrayList<SingleUnitImpl>	MeasureUnitImpl.java	/icu4j-core/src/com/ibm/icu/impl/units	line 227	Java Problem

Reply:
(I recommend simply not changing this line - it seems fine as it was, and Java 7 compatible?)

Reply:
I reopened this in Eclipse to take another look, and this error is no longer showing up. I don't know enough about Eclipse and Java to speculate what might have been different when I saw this error before.

I'll mark as resolved.

Reply:
Turns out the error was indeed a problem, and this line needs to be changed back.